### PR TITLE
Material components

### PR DESCRIPTION
**Pull request by mikehern**
- Added color theme.
- Fixed previous bug preventing new custom events from being added to timeline.
- Replaced default buttons with Material Design theme.
- Replaced text input fields with Material Design theme.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{
+}
```
Comment (kfang1220) [inline .vscode/settings.json]: This file won't do anything right?

File: client/src/SearchBar.jsx [hunk 2]
```diff
@@ -9,24 +13,41 @@ const SearchBar = (props) => {
     }
   };
 
+  const style = {
+    marginTop: 12,
+  }
+
```
Comment (chalkera) [inline client/src/SearchBar.jsx]: This should be in the styles

File: client/src/TimelineLookUp.jsx [hunk 3]
```diff
@@ -14,20 +19,37 @@ const TimelineLookUp = (props) => {
     }
   };
 
+  const styles = {
```
Comment (chalkera) [inline client/src/TimelineLookUp.jsx]: Can you put this in the style sheets

**Review by chalkera — COMMENTED** [review]
Would like to see a few changes if possible

**Review by kfang1220 — COMMENTED** [review]
Looks good to me

**Comment by mikehern** [comment]
This specific component library doesn't allow for clean className override. Axed some of these customizations, for now. (The tradeoffs of using someone else's work...)

**Review by kfang1220 — APPROVED** [review]
Looks good to me, just axed a bunch of CSS

**Pull request merged**